Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -1660,17 +1660,21 @@ bool CStaticFunctionDefinitions::GetPedControlState(CClientPed& Ped, const char*
     return false;
 }
 
-bool CStaticFunctionDefinitions::GetPedAnalogControlState(CClientPed& Ped, const char* szControl, float& fState)
+bool CStaticFunctionDefinitions::GetPedAnalogControlState(CClientPed& Ped, const char* szControl, float& fState, bool bRawInput)
 {
     if (Ped.GetType() == CCLIENTPLAYER)
     {
         CControllerState cs;
         Ped.GetControllerState(cs);
         bool         bOnFoot = (!Ped.GetRealOccupiedVehicle());
         unsigned int uiIndex;
+
+        if (bRawInput)
+            cs = Ped.m_rawControllerState;

```

Comment:
If `bRawInput` is `true` then line 1668 is useless. I would suggest this:

```suggestion
            cs = Ped.m_rawControllerState;
        else
            Ped.GetControllerState(cs);
```